gdnative-derive/src/native_script.rs

```diff
@@ -144,14 +144,14 @@ fn parse_derive_input(input: &DeriveInput) -> Result<DeriveData, syn::Error> {
 
     let ident = input.ident.clone();
 
-    let inherit_attr = input
-        .attrs
-        .iter()
-        .find(|a| a.path.is_ident("inherit"))
-        .ok_or_else(|| syn::Error::new(span, "No \"inherit\" attribute found"))?;
+    let inherit_attr = input.attrs.iter().find(|a| a.path.is_ident("inherit"));
```

Comment:
Note: this was changed by `cargo fmt`.
I don't know what level it needs to split lines.